translation/en/RedirectedResources/assets/abdata/adv/scenario/c07/00/72/translation.txt

```diff
@@ -1,5 +1,5 @@
-………………=.........
-…………=......
+﻿………………=...
+…………=...
 ……=...
```

Comment:
Is this intended?

Reply:
yes, those massive amounts of dots get simplified down quite a bit as they just don't make sense to nuke all over in english